templates/web/bromley/header_logo.html

```diff
@@ -62,7 +62,11 @@
         <!-- Breadcrumb --><!-- googleoff:all -->
         <ul>
             <li class="first-child"><a href="https://www.bromley.gov.uk">Home</a></li>
+          [% IF bodyclass == 'waste' %]
```

Comment:
Is the ` OR (problem AND problem.cobrand_data == 'waste')` bit below needed here as well?

Reply:
You can't see this breadcrumb on a report page as it's a map page, so it doesn't matter, but it would keep it consistent IF wise, I guess, so might as well, yeah.